phpBB/config/default/container/services_feed.yml

```diff
@@ -13,6 +13,7 @@ services:
             - '@auth'
             - '@dispatcher'
             - '%core.php_ext%'
+            - '@language'
```

Comment:
I would prefer an alphabetical order in the .yml and in the `%` entries at the end, e.g.: https://github.com/phpbb/phpbb/blob/master/phpBB/config/default/container/services.yml which will make it easier to find the services.

Reply:
Thanks for the review, I fix it